eva-pipeline/src/main/java/embl/ebi/variation/eva/pipeline/annotation/generateInput/VariantAnnotationItemProcessor.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2015-2016 EMBL - European Bioinformatics Institute
```

Comment:
Fixed all files in next PR. I've added a copyright profile as described here: https://www.jetbrains.com/help/idea/2016.2/generating-and-updating-copyright-notice.html
Like this copyright is added automatically when you create a new file